### PR TITLE
Fix select currency issues after edit offer

### LINKED ISSUE #1565: Bug with fixed currency at edit offer
After  a user has clicked edit offer the currency will stick in create offer views as well as in the edit offer view.
To reproduce create 2 offers with fiat and an altcoin. Then go to edit offer at both and back to create offer. Switch payment method at create offer. The currency does not change anymore. At the edit offer it shows the same currency for both offers. 

See also: https://bisq.community/t/create-buy-offer-wrong-currency/5573/11

### PR DESCRIPTION
**Pull request by ripcurlx**
Fixes #1565.

### CODE REVIEW

File: src/main/java/bisq/desktop/main/portfolio/editoffer/EditOpenOfferDataModel.java [hunk 1]
```diff
@@ -49,14 +49,21 @@
...
+        super(openOfferManager, btcWalletService, bsqWalletService, preferences, user, keyRing, p2PService,
+                priceFeedService, filterManager, accountAgeWitnessService, tradeWalletService, feeService, formatter);
     }
 
     public void initWithData(OpenOffer openOffer) {
         this.openOffer = openOffer;
         this.initialState = openOffer.getState();
         this.paymentAccount = user.getPaymentAccount(openOffer.getOffer().getMakerPaymentAccountId());
+        // select the current offer payment account as default payment account
+        preferences.setSelectedPaymentAccountForCreateOffer(paymentAccount);
```
Comment (ManfredKarrer) [inline src/main/java/bisq/desktop/main/portfolio/editoffer/EditOpenOfferDataModel.java]: Is that needed in the base class or should that be only called in the createOffer sub class? If so, maybe better to overwrite initWithData there.

File: src/main/java/bisq/desktop/main/portfolio/editoffer/EditOpenOfferView.java [hunk 2]
```diff
@@ -131,9 +131,9 @@ protected void deactivate() {
     ///////////////////////////////////////////////////////////////////////////////////////////
 
     public void initWithData(OpenOffer openOffer) {
+        model.initWithData(openOffer);
```
Comment (ManfredKarrer) [inline src/main/java/bisq/desktop/main/portfolio/editoffer/EditOpenOfferView.java]: I would prefer if we don't have that call before the super call. Carries risky dependencies of order of calls (see my comment in other commit - maybe that solves the issue without changing here the order?)

**Review by ManfredKarrer — CHANGES_REQUESTED** [review]

File: src/main/java/bisq/desktop/main/portfolio/editoffer/EditOpenOfferDataModel.java (hunk 1 above)
Comment (ripcurlx) [reply src/main/java/bisq/desktop/main/portfolio/editoffer/EditOpenOfferDataModel.java]: Actually this is already the edit offer sub class. The naming of the method is misleading, I'll rename it to a generic one.

**Comment by ripcurlx** [comment]
@ManfredKarrer: I think I somehow f** up this PR. I'll delete it an re-open a new one with a different strategy that makes it even easier to solve this issue 😄

**Pull request closed without merging**